StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentMethodType.swift

```diff
@@ -289,6 +378,40 @@ extension PaymentSheet {
             let supports = Set(requirements).isSubset(of: fulfilledRequirements)
             return supports
         }
+
+        /// DRY helper method
+        static func supports(
+            paymentMethod: STPPaymentMethodType,
+            requirements: [PaymentMethodTypeRequirement],
+            configuration: PaymentSheet.Configuration,
+            intent: Intent,
+            supportedPaymentMethods: [STPPaymentMethodType]
+        ) -> Bool {
```

Comment:
How is this different from the `supports` method above?  When would someone use one vs. the other?

Reply:
Arg, yeah they are really similar, the one above seems slightly modified to use the dynamic PM stuff. As it doesn't take a `STPPaymentMethodType` since a dynamic payment method cannot be expressed as a `STPPaymentMethodType`.

I tried to consolidate these into one but it gets pretty ugly mixing the dynamic `PaymentSheet.PaymentMethodType` with `STPPaymentMethodType`. So I just renamed these in an attempt to be clearer and added doc comments.